src/components/ObjectFormField.vue

```diff
@@ -96,7 +96,7 @@
             </template>
             <!-- default -->
             <template v-else>
-                {{ valueLabel || value }}
+                {{ valueLabel || value ? value : '-' }}
```

Comment:
could we make the - grey instead of black?

Reply:
Fixed it with c7c2bc63549312aca52d4df28f30cbb98f623b48